### PR TITLE
Add GitHub community documentation

### LINKED ISSUE #368: Community documentation
We should add community-related documentation to the repository to help guide contributors. This is the current state:

image

### PR DESCRIPTION
**Pull request by martincostello**
Resolves #368.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```
Comment (brainmurphy) [inline CODE_OF_CONDUCT.md]: Love it! ❤️

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+_Explain what you expected to happen._
+
+### Actual behaviour
+
+_Explain what actually happened. If an exception occurred, please include a stack trace if available._
+
+### Steps to reproduce
+
+_A concise, repeatable, example of how to illustrate the issue._
```
Comment (adammorr) [inline .github/ISSUE_TEMPLATE.md]: Being picky and possibly wrong but should this be _A concise and repeatable example of how to illustrate the issue_?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -22,15 +22,15 @@ If you find a bug, have a feature request or even want to contribute an enhancem
 
 ### General Contribution Guidelines:
 We follow the standard GitHub fork & pull request approach to open source collaboration.
-You can find an awesome description on how this works on the [gun.io blog](https://gun.io/blog/how-to-github-fork-branch-and-pull-request/).
+You can find an awesome description on how this works on the [gun.io blog](https://gun.io/blog/how-to-github-fork-branch-and-pull-request/ "How to GitHub: Fork, Branch, Track, Squash and Pull Request").
 
 In a nutshell:
 
-0. Fork it.
```
Comment (brainmurphy) [inline .github/CONTRIBUTING.md]: 😉

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+name: Bug report
+about: Create a bug report to help us improve JustSaying
+
+---
+
+**Describe the bug**
+_A clear and concise description of what the bug is._
+
+**Steps To reproduce**
+_A concise, repeatable, example of how to reproduce the issue._
```
Comment (adammorr) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Same as my last comment, should this be; _A concise and repeatable example of how to illustrate the issue_?

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (martincostello) [reply .github/ISSUE_TEMPLATE.md]: This is mainly copied as-is from the GitHub UI's master template, so happy to amend.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (martincostello) [reply .github/ISSUE_TEMPLATE/bug_report.md]: As above.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (adammorr) [reply .github/ISSUE_TEMPLATE.md]: OK then let's leave it. It's probably just how I was reading it in my head
